packages/aggregator/src/__tests__/analytics/filterAnalytics.test.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { filterAnalytics } from '../../analytics/filterAnalytics';
+
+const analytics = [{ value: 1 }, { value: 2 }, { value: 3 }];
+
+describe('filterAnalytics()', () => {
+  describe('empty filter', () => {
+    const testData = [
+      ['i', []],
+      ['ii', [undefined]],
+      ['iii', [{}]],
```

Comment:
We can use descriptive names here. In order: 'empty array', 'undefined', 'empty object'